dashdotdb/__init__.py

```diff
@@ -22,20 +23,33 @@
                     f'{DATABASE_PORT}/'
                     f'{DATABASE_NAME}')
 
-
 class DashDotDb(App):
+    def liveness(self):
+        pass
+
+    def readiness(self, db):
+        try:
+            pass # Have to figure this out
```

Comment:
Ideas here are welcome

Reply:
I like the idea of doing a database connection/ping check.

I think this shows a pretty simple way of doing it: https://stackoverflow.com/questions/30307658/flask-sqlalchemy-check-if-database-server-is-responsive